public/js/education.js

```diff
@@ -0,0 +1,47 @@
+$(document).ready(function(){
```

Comment:
You didn't need to create a new JS file. Simply change the button and add it to the form JS